<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Current</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="current_input">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Time</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="time_input">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="set_current_button">
            <property name="text">
             <string>Set current</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="set_time_button">
            <property name="text">
             <string>Set time</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="on_off_button">
            <property name="text">
             <string>Turn on/off</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Current:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="current_display">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="prepare_repeat_button">
          <property name="text">
           <string>Prepare repeat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="start_triggered_button">
          <property name="text">
           <string>start triggered repeat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="start_untriggered_button">
          <property name="text">
           <string>start untriggered repeat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="start_bus_triggered_button">
          <property name="text">
           <string>start software triggered repeat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="send_trigger_button">
          <property name="text">
           <string>send trigger</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="abort_button">
          <property name="text">
           <string>Abort repeat</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
